Update internal docs (#427)
neuro-inc/cookiecutter-neuro-project: {{cookiecutter.project_slug}}/HELP.md

### Setup development environment 

```shell
neuro-flow build myimage
```

> * Several files from the local project are uploaded to the platform storage (namely, `requirements.txt`,  `apt.txt`, `setup.cfg`).

Reviewer: Pls, update that the entire `context` folder is uploaded.

neuro-flow kill jupyter
```

> * The job with Jupyter Notebooks is terminated. The notebooks are saved on the platform storage.

Reviewer: we could preserve it, neuro-flow could do that
```suggestion
* The job with Jupyter Notebooks is terminated. The notebooks are saved on the platform storage. You may run `neuro-flow download notebooks` to download them to the local `notebooks/` directory.
```

### Help

```shell 
neuro-flow ps
```

* The list of all available template jobs is printed along with their statuses.

## Data

### Uploading to the Storage via Web UI

On local machine, run `neuro-flow run filebrowser` and open the job's URL on your mobile device or desktop.
Through a simple file explorer interface, you can upload test images and perform file operations.

### Uploading to the Storage via CLI

On local machine run `neuro-flow mkvolumes`. This commands creates storage folders for all defined volumes. You only need to run this once.

> After storage folders have been created run `neuro-flow upload data` from local machine as well. This command pushes local files stored in ./data into storage:{{ cookiecutter.project_slug }}/data mounted to your development environment's /project/data.

Reviewer: Let's add a bit more info:
```suggestion
After storage folders have been created run `neuro-flow upload data` from the local machine as well. This command pushes local files stored in ./data into storage:{{ cookiecutter.project_slug }}/data mounted to your development environment's /project/data.

You can upload (or download) every folder for which the `local` parameter is specified in [live.yml file](./.neuro/live.yml).
```

### Uploading data to the Job from Google Cloud Storage

Google Cloud SDK is pre-installed on all jobs produced from the Base Image.

> Please refer to the [documentation](https://docs.neu.ro/toolbox/accessing-object-storage-in-gcp) explaining how to start using GCP with the template.

Reviewer: these docs will be updated to refer `neuro-extras` usage later?

Author: Yes, absolutely. It is in the plans: as soon as `neuro-flow` supports multiple volumes/envvars passing to custom actions

### Uploading data to the Job from AWS S3

AWS CLI is pre-installed on all jobs produced from the Base Image.

Please refer to the [documentation](https://docs.neu.ro/toolbox/accessing-object-storage-in-aws) explaining how to start using AWS with the template.

### Run development job

If you want to debug your code on GPU, you can run a sleeping job via `neuro-flow run remote_debug`, which will also open a shell to the job. You can also see job logs via `neuro-flow logs remote_debug`. The job forwards your local port 2211 to its port 22for remote debugging.
Please find instructions on remote debugging via PyCharm Pro in the [documentation](https://neu.ro/docs/remote_debugging_pycharm). 

> Please don't forget to kill your job via `make kill remote_debug` not to waste your quota!   

Reviewer: ```suggestion
Please don't forget to kill your job via `neuro-flow kill remote_debug` not to waste your quota!   
```
